.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,5 @@
-<!--  Thanks for sending a pull request!  Here are some tips for you:
-1. If this is your first time, please read our contributor guidelines: https://github.com/Code-Hex/vz/blob/master/CONTRIBUTING.md
-2. Please create a new issue before creating this PR. However, You can continue it without creating issues if this PR fixes any documentations such as typo.
-3. Do not send Pull Requests for large (150 ~ lines) code changes. If so, I am not motivated to review your code. Basically, I write the code.
+<!--  Thanks for sending a pull request!
```

Comment:
This change and the removal of  `FUNDING.yml` and `CONTRIBUTING.md` are not related to the module renaming.

Reply:
Those files were specific to https://github.com/Code-Hex/vz  and not appropriate for https://github.com/lima-vm/vz

Reply:
Yeah I understand, and I agree they should be removed/rewritten. For clarity, I would have put these changes in a separate commit with the explanation you just added.